### PR TITLE
[AIFBR-10082]added TM model metadata

### PR DESCRIPTION
**Pull request by aakashuipath**

### CODE REVIEW

File: metadata/TaskMining__1__metadata.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    "inputType": "JSON",
+    "displayName": "TMAnalyzerModel",
+    "memory": 0,
+    "mlPackageLanguage": "PYTHON37",
+    "projectId": "[project-id]",
+    "stagingUri": "[staging-uri]",
+    "projectName": "UiPath Task Mining",
+    "projectDescription": "UiPath Task Mining models to analyze captured data",
+    "tenantName": "UiPath",
+    "minAIFabricVersion": "v20.10.1.2",
```
Comment (vivekjhauipath) [inline metadata/TaskMining__1__metadata.json:20]: change this

File: metadata/TaskMining__1__metadata.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    "languageVersion": 0,
+    "version": 1,
```
Comment (vivekjhauipath) [inline metadata/TaskMining__1__metadata.json:22]: can you check with TM team whether the model works with DU v2 baseimage?

**Review by vivekjhauipath — CHANGES_REQUESTED** [review]
Add another field:
version: 1

This denotes package version and should match the one in file name

File: metadata/TaskMining__1__metadata.json (hunk 1 above)
Comment (aakashuipath) [reply metadata/TaskMining__1__metadata.json:20]: done

File: metadata/TaskMining__1__metadata.json (hunk 2 above)
Comment (aakashuipath) [reply metadata/TaskMining__1__metadata.json:22]: okay I will check in the channel.

**Comment by aakashuipath** [comment]
@vivekjhauipath  added version field in the metadata and changed minversion

**Review by vivekjhauipath — APPROVED** [review]
LGTM

**Review by SahilChaudhary25 — APPROVED** [review]
LGTM

**Pull request merged**